rfcs/text/0012-orchestrator-field-set.md

```diff
@@ -19,9 +19,68 @@ add a new schema rather than change existing material.
 
 ## Fields
 
-<!--
-Stage 1: Describe at a high level how this change affects fields. Which fieldsets will be impacted? How many fields overall? Are we primarily adding fields, removing fields, or changing existing fields? The goal here is to understand the fundamental technical implications and likely extent of these changes. ~2-5 sentences.
--->
+The proposed change adds six fields, as described below:
+
+```
+---
+- name: orchestrator
+  title: Orchestrator
+  group: 2
+  short: Fields relevant to container orchestrators.
+  description: >
+    Fields that describe the resources which container orchestrators manage or
+    act upon.
+  type: group
+  fields:
+    - name: cluster
+      level: core
+      type: keyword
+      example: production
+      description: >
+        Orchestrator cluster name.
+
+    - name: type
+      level: core
+      type: keyword
+      example: Nomad, Kubernetes
+      description: >
+        Orchestrator cluster type.
+
+    - name: namespace
```

Comment:
Cloud Foundry has `organization`s and `space`s.

Wdyt about having an `organization` field for multi-tenant clusters, and `namespace` for logical separations inside the cluster/organization?

Reply:
Sounds reasonable to me, I'll add this 👍